packages/shared/src/components/modals/common/ModalFooter.tsx

```diff
@@ -16,8 +16,8 @@ export function ModalFooter({
   justify = Justify.End,
   view,
 }: ModalFooterProps): ReactElement {
-  const { activeView } = useContext(ModalPropsContext);
-  if (view && view !== activeView) {
+  const { activeView, isDrawer } = useContext(ModalPropsContext);
```

Comment:
Footer is not needed on Drawers.

Reply:
It is highly likely that it won't happen. Since the footer has a top border and is at the bottom-most part. It would clash with the built-in "Close" button.

Reply:
Also wondered if we could just make it dynamic based on high level component or perhpas just not render modalFooter at all.
But might miss some design edge-cases?